### PR TITLE
Modular computer renaming

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Basically, I took every description/name/output and replaced "NTNet" to "SCiPnet", "NTOS" to "SCPOS", and "NTNRC" to "SCPRC". I also changed some of the comments to refer to the system by SCP names, but not individual components (that is, comments referring to specific defines aren't changed)

## Why It's Good For The Game

The less "Nanotracen" a player sees, the better.

## Changelog

:cl:
fix: Modular computers and their programs/components refer to SCP instead of NT.
/:cl:

### CODE REVIEW

**Review by hry-gh — APPROVED** [review]
looks good + doesn't break all the computers

File: code/modules/modular_computers/file_system/programs/generic/news_browser.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/computer_file/program/newsbrowser
 	filename = "newsbrowser"
-	filedesc = "NTNet/ExoNet News Browser"
+	filedesc = "SCiPnet/ExoNet News Browser"
```
Comment (TheTitanicEngineer) [inline code/modules/modular_computers/file_system/programs/generic/news_browser.dm]: Recommend removing ExoNet as well

Comment (aidan-bowman) [reply code/modules/modular_computers/file_system/programs/generic/news_browser.dm]: done

**Comment by hry-gh** [comment]
let's go

**Pull request merged**